docker-images/baleine/Dockerfile

```diff
@@ -1,5 +1,28 @@
 FROM ubuntu:latest
 
+# Install necessary binaries for tutorials
+RUN apt update -y && apt upgrade -y && apt install kmod iw iputils-ping openssh-client openssh-server bash netcat iproute2 net-tools tcpdump dnsutils netcat nano vim -y
+
+# Setup Bash prompt to not get lost when in interactive mode
+RUN echo "PS1='[CONTAINER] ${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '" | tee /root/.bashrc > /dev/null
+
+#Setting up OpenSSH server
+RUN sed -ri 's/^#?PermitRootLogin\s+.*/PermitRootLogin yes/' /etc/ssh/sshd_config
+RUN sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config
+RUN sed -i 's/#Port 22/Port 2222/g' /etc/ssh/sshd_config
+RUN sed -i 's/#ListenAddress 0.0.0.0/ListenAddress 0.0.0.0/g' /etc/ssh/sshd_config
+RUN echo "auth sufficient pam_permit.so" | tee -a /etc/pam.d/sshd >> /dev/null
+
+RUN mkdir -p /run/sshd
```

Comment:
this one feels odd too; why is that needed ?

Reply:
Without this folder in the container, `sshd` crashes.